docs/layers.md

```diff
@@ -1595,34 +1595,34 @@ The `hgv` property indicates general truck heavy goods vehicle truck access, val
+
+A basic Time Domain is the combination of a Starting Date and a Time Duration with the following notation: [(Starting Date) {Time duration}]. For example, `[(M5d1){d1}]` means: Starting Date: any year, month 5 (May), day 1st, at 0:00 a.m. Duration: 1 complete day (i.e., 24 hours or 1440 minutes).
+
+Starting Dates and time intervals are defined by means of a set of graphical symbols allowing the description of years, months, weeks, days, and so on down to the smallest time unit, which is the second. The symbols have to be organized in a sequential order starting with the longest time unit. Attached to a starting date, the interval constitutes a basic Time Domain. Without a starting date, it just indicates a duration. If the very first time type code in duration section is preceded by a minus sign, it means that the duration is counted in the reverse order. The following list describes starting date and time interval unit types:
+* `ynn` - Year. Defines either a particular nnnn year in a starting date; or a duration of nn years. For example, [(y1991M11d14h5m30s19){y1}] means from 14 November 1991, 5:30:19 a.m. to 14 November 1992, 5:30:19 a.m. If there is no identical calendar date in the year in question, which occurs only for 29 February, “plus 1 year” leads to 28 February of the following year. Notice that {y1 = {M12}
+* `Mnn` - Month. Defines either a particular month (1 to 12) within a particular year, or any year when no `y` information is given in starting date; or a duration of nn months. For example, [(y1991M11d14h5m30s19) {M3}] means from 14 November 1991, 5:31:19 a.m. to 14 February 1992, 5:30:19 a.m.
+* `wnn` - Week. Defines either a week (1 to 53) within a previously defined year, or any year when no `y` information is given in a starting date; or a duration of nn weeks, i.e., nn*7 days. For example, [(y1991M11d14h5m30s19) {w2}] means from 14 November 1991 at 5:30:19 a.m. to 28 November 1991, 5:30:19 a.m. Notice that {w1} = {d7}
+* `dnn` - Day. Defines either a particular day (1 to 28, 29, 30, 31) within a particular month if previously defined with the `M` format in a starting date; or a duration of nn days, i.e., nn*24 hours. For example, [(y1991M11d14h5m30s19) {d2}] means from 14 November 1991 at 5:30:19 a.m. to 16 November 1991, 5:30:19 a.m. Notice that {d1} = {h24}
+* `tn` - Day. Only in starting date defines a particular weekday (1:Sunday, 2:Monday, 3:Tuesday, 4:Wednesday, 5:Thursday, 6:Friday, 7:Saturday) in a previously (if any) defined week. For example, (M5t2) means each Monday in the fifth month (May) of any year, at 0:00:00 a.m.
+* `fxn` - Day. Only in starting date defines a particular weekday in a previously (if any) defined month, with the following rules: n is used as in the “t” format with the same domain of values, 1:Sunday up to 7:Saturday. For “x” one of the following values has to be substituted: 1:first, 2:second, 3:third, 4:fourth, 5:fifth. For example: (...f12) means the first Monday at 0:00:00 a.m.
```

Comment:
I don't follow this example

Reply:
fxn:  x=1 - first, n=2 - Monday, so f12 means first Monday
 f22 would mean second Monday

Reply:
I understand now, thanks